lms/djangoapps/instructor/views/instructor_dashboard.py

```diff
@@ -106,6 +107,9 @@ def show_analytics_dashboard_message(course_key):
 @cache_control(no_cache=True, no_store=True, must_revalidate=True)
 def instructor_dashboard_2(request, course_id):
     """ Display the instructor dashboard for a course. """
+    start_time = time.time()
+    if course_id == 'course-v1:HarvardX+CS50+X':
+        log.info('Investigating log: Start of instructor_dashboard_2')
```

Comment:
We can remove this log. It does not provide any useful information

Reply:
Done